src/Header/Account/Account.jsx

```diff
@@ -133,6 +146,8 @@ export default class Account extends Component {
 					}.bind(this)
 				);
 		}
+
+		document.addEventListener("click", this.handleMegaMenuClick);
```

Comment:
_generally_ in the react world, wherever you add an event listener, you need to remove it if the component unmounts. it's not necessarily always an issue, cos possibly the component doesn't ever unmount. i'm going to test something out with this.